src/modules/render.js

```diff
@@ -0,0 +1,35 @@
+const ul = document.getElementById('meal-list');
+const renderMeals = (mealsData) => {
+  console.log(mealsData);
+  const { meals } = mealsData;
+  console.log(meals);
+
+  meals.forEach((mealItem) => {
+    const li = document.createElement('li');
+    li.className = 'meal-item';
+    const img = document.createElement('img');
+    img.src = mealItem.strMealThumb;
+    img.className = 'meal-image';
+    li.appendChild(img);
+    const div = document.createElement('div');
+    div.className = 'meal-meta';
+    li.appendChild(div);
+    const p = document.createElement('p');
+    p.className = 'meal-title';
+    p.innerText = mealItem.strMeal;
+    div.appendChild(p);
+    const likeBtn = document.createElement('button');
+    likeBtn.className = 'icn-btn';
+    div.appendChild(likeBtn);
+    const likeIcon = document.createElement('i');
+    likeIcon.className = 'far fa-heart';
+    likeBtn.appendChild(likeIcon);
+    const commentBtn = document.createElement('button');
```

Comment:
Great work on implementing the Comments button, but I think you forgot to add the reservations button too

'Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.'

Reply:
Resolved